docs/docker.md

```diff
@@ -24,8 +24,9 @@ root@c278c88dd552:/examples# mpirun -np 4 -H localhost:4 python keras_mnist_adva
 ```
 
 You may notice that this command does not have a few options recommended in other parts of documentation: 
```

Comment:
rephrase this now to "This command does not have options recommended in other parts of the documentation. `-bind-to none -map-by slot -x NCCL_DEBUG=INFO` options are already ...